kahuna/public/js/components/gr-downloader/gr-downloader.html

```diff
@@ -47,9 +47,8 @@
           aria-label="Cannot download">
         <gr-icon-label
             gr-icon="file_download"
-            class="download-icon-label"
-            gr-tooltip="Downloading non-free images requires a lease"
-        >
+            class="cannot-download-icon-label"
```

Comment:
this works, but did you consider using the `side-padded` class that the other buttons use to add padding? it would mean one less CSS rule to maintain for just this version of the button